cartography/intel/gcp/storage.py

```diff
@@ -0,0 +1,114 @@
+import logging
+from collections import namedtuple
+
+import googleapiclient.discovery
+from oauth2client.client import ApplicationDefaultCredentialsError
+from oauth2client.client import GoogleCredentials
+
+from cartography.intel.gcp import compute
+logger = logging.getLogger(__name__)
+
+
+def get_bucket_metadata(storage, bucket):
+    """
+    Retrieves metadata about the given bucket
+
+    :type storage: A storage resource object
+    :param storage: The storage resource object created by googleapiclient.discovery.build()
+
+    :type bucket: str
+    :param bucket: Google Cloud Bucket name
+
+    :rtype: dict
+    :return: Metadata for specified bucket
+    """
+    try:
+        req = storage.buckets().get(bucket=bucket)
+        res = req.execute()
+        return res
+    except HttpError as e:
+        reason = compute._get_error_reason(e)
+        if reason == 'notFound':
+            logger.debug(
+                ("The bucket %s was not found - returned a 404 not found error."
+                 "Full details: %s"), bucket, e, )
+            return None
+        elif reason == 'forbidden':
+            logger.debug(
+                ("You do not have storage.bucket.get access to the bucket %s. "
+                 "Full details: %s"), bucket, e, )
+            return None
+        else:
+            raise
+
+
+def list_buckets(storage, project):
```

Comment:
I would rename to get_buckets just so for consistency with the other get* functions.

Reply:
Done! Nice catch, sorry for the delay.

Reply:
I need to review neo4j syntax for the load* function, but I should be fine, thanks! If I have any specific questions I'll reach out within the next few days.